core/src/main/java/smile/regression/TreeShapImportance.java

```diff
@@ -0,0 +1,735 @@
+package smile.regression;
```

Comment:
Can you please format the code by our standard? IntelliJ has shortcut to do so. Thanks.

Reply:
I use eclipse, will try the code format feature in Eclipse.  Do you have the code format from IntelliJ to import, so I might could use it in Eclipse to format the code? Thanks